apps/docs/docs/faq.md

```diff
@@ -25,3 +25,69 @@ This is a good option if you are using TypeScript/JavaScript and want to interac
 - [Boilerplate](https://github.com/bandada-infra/boilerplate)
 
 This is a good option if you want to quickly create a Bandada project because you can fork it, clone it or use it as a template.
+
+## What is the difference between Bandada and Semaphore?
+
+When it comes to [Bandada](https://bandada.pse.dev/) and [Semaphore](https://semaphore.pse.dev/), you might find yourself wondering how these two platforms stack up against each other. Both are designed to streamline and enhance different aspects of project management and automation, but they cater to distinct needs and offer unique features. Here’s a comprehensive comparison to help you understand how they differ.
+
+### **Bandada: An Overview**
+
+**Bandada** is an infrastructure to manage privacy-preserving groups. It also provides antisybil mechanisms by using credential groups so that you can only join a group if you meet a specific criteria.
+
+#### **Key Features of Bandada:**
+
+- **Intuitive Dashboard:** Provides a user-friendly interface for easy navigation and task management.
+- **Customizable Workflows:** Allows you to tailor workflows according to your team's specific needs.
+- **Integration Capabilities:** Seamlessly integrates with other tools and platforms to enhance productivity.
+- **Real-Time Collaboration:** Facilitates real-time communication and collaboration among team members.
+
+#### **Target Audience:**
+
+Bandada is ideal for:
+- **Teams and Organizations:** Looking for a robust project management solution.
+- **Collaborative Work Environments:** Where real-time updates and integration with various tools are crucial.
+- **Customization Enthusiasts:** Who need a platform that can be tailored to their specific workflow requirements.
```

Comment:
This is not very accurate. Bandada is for DAOs, developers, teams and organizations that would like to manage and work with privacy preserving groups.